dataland-frontend/src/components/resources/frameworkDataSearch/FrameworkDataSearchResults.vue

```diff
@@ -1,71 +1,67 @@
-            <template #header>
-              <span class="uppercase">PERM ID</span>
-              <i
-                class="material-icons pl-2"
-                aria-hidden="true"
-                title="Perm ID"
-                v-tooltip.top="{
-                  value:
-                    'Permanent Identifier (PermID) is a machine readable identifier that provides a unique reference ' +
-                    'for data items including organizations, instruments, funds, issuers and people. You can search and verify an id at permid.org/search',
-                  class: 'd-tooltip',
-                }"
-                >info</i
-              >
-            </template>
-            <template #body="{ data }">
-              {{ data.permId ? data.permId : "Not available" }}
-            </template>
-          </Column>
-          <Column field="companyInformation.sector" header="SECTOR" :sortable="true" class="d-bg-white w-2" />
-          <Column field="companyInformation.headquarters" header="LOCATION" :sortable="true" class="d-bg-white w-2">
-            <template #body="{ data }">
-              {{ buildLocationString(data.companyInformation.headquarters, data.companyInformation.countryCode) }}
-            </template>
-          </Column>
-          <Column field="companyId" header="" class="d-bg-white w-1 d-datatable-column-right">
-            <template #body="{ data }">
-              <router-link :to="getRouterLinkTargetFrameworkInt(data)" class="text-primary no-underline font-bold"
-                ><span> VIEW</span> <span class="ml-3">></span>
-              </router-link>
-            </template>
-          </Column>
-        </DataTable>
-        <div class="d-center-div text-center px-7 py-4" v-else>
-          <p class="font-medium text-xl">Sorry! Your search didn't return any results.</p>
-          <p class="font-medium">Try again please!</p>
-        </div>
-      </div>
+        </template>
+        <template #body="{ data }">
+          {{ data.permId ? data.permId : "Not available" }}
+        </template>
+      </Column>
+      <Column field="companyInformation.sector" header="SECTOR" :sortable="true" class="d-bg-white w-2" />
+      <Column field="companyInformation.headquarters" header="LOCATION" :sortable="true" class="d-bg-white w-2">
+        <template #body="{ data }">
+          {{ buildLocationString(data.companyInformation.headquarters, data.companyInformation.countryCode) }}
+        </template>
+      </Column>
+      <Column field="companyId" header="" class="d-bg-white w-1 d-datatable-column-right">
+        <template #body="{ data }">
+          <router-link :to="getRouterLinkTargetFrameworkInt(data)" class="text-primary no-underline font-bold"
+            ><span> VIEW</span> <span class="ml-3">></span>
+          </router-link>
+        </template>
+      </Column>
+    </DataTable>
+    <div class="d-center-div text-center px-7 py-4" v-else>
```

Comment:
I will push a commit where I centralize the "d-center-div" css-class and do a small change on the AuthenticationWrapper.vue.
Please review this commit then.